sites/footers/footers-deprecated-en.html

```diff
@@ -0,0 +1,51 @@
+---
+{
+	"title": "Content page including complete footer (deprecated)",
+	"language": "en",
+	"altLangPage": "footers-deprecated-fr.html",
+	"secondlevel": false,
+	"dateModified": "2022-03-18",
+	"share": "true",
+	"footerDeprecated": "true"
```

Comment:
@duboisp What's your take on using a ``footerDeprecated`` variable and having the footer include (``sites/footers/includes/footer.html``) check on ``page.footerDeprecated``)? It'd be less repetitive than adding yet another layout file.

If you'd still prefer going with a dedicated layout file I can make further revisions to add one in.

Reply:
Let discuss at our next technical review or our next office hours.

May be we should have something that will allow us to deprecate it again because we know that DTO are currently looking at redesigning the site footers section.

Reply:
Just throwing this out there, but maybe that could be achieved with more specific variable/file names? For example, using a naming scheme along the lines of ``old[Topic][Change]`` or ``old[Topic][YYYYMMDD]`` for variables and file names? If one of those naming schemes were to be applied to this PR, we'd end up with ``oldFooterColcountXs2`` or ``oldFooter20220510``.

Reply:
Ended up scrapping the ``footerDeprecated`` variable and doing something very similar to #1903 and #1962's deprecated page samples.